pkgs/tools/cd-dvd/unetbootin/default.nix

```diff
@@ -1,13 +1,13 @@
-{ stdenv, fetchurl, makeWrapper, qt4, utillinux, coreutils, which, p7zip, mtools, syslinux }:
+{ stdenv, fetchurl, makeWrapper, qt4, utillinux, coreutils, which
+, p7zip, mtools, syslinux }:
 
-let version = "608"; in
-
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   name = "unetbootin-${version}";
+  version = "613";
 
```

Comment:
I meant "where" literally. :-) Why change from `let` to `rec` like this? I don't see the need.